docs/sources/configuration/perception.rst

```diff
@@ -0,0 +1,51 @@
+Perception
+============
+
+The collision avoidance depends mostly on the Configuration > Perception dialog where the connection and description of either a laserscan sensor or an obstacle map message is configured.
+
+.. image:: ../data/cpanel5.png
+   :width: 800px
+   :alt: Configuration > Perception page: The laserscan or list of obstacles is configured here.
+
+- **Laserscan sensor**: The Supervisor can handle data points from one 2-dimensional laser (LIDAR) scanner.
+
+  * **Display name**: A user-specified name can be provided so that diagnostic information can quickly be understood.
+
+  * **ROS Topic**: The ROS message name, topic type, quality of service and Signal Rate must all be specified so that the Supervisor can subscribe to the sensor data.
```

Comment:
Supevisor needs to subscribe to the  ROS message published by the LIDAR, so this topic name (plus type, quality-of-service, and expected signal rate) need to match the sensor's output. *ros2 topic * with options *list, info, hz" can be useful to obtain this information.